src/components/layouts/table/table.config.js

```diff
@@ -0,0 +1,78 @@
+module.exports = {
+    order: 6,
+    status: "ready",
+    context: {
+        label: "Our Party",
+        caption: "Members of the party.",
```

Comment:
I suppose it isn't clear that the label isn't visible and is just used as an accessible name for the `<div>` around the table. For someone who may not understand the structure, it may also not be clear what the purpose or difference between the caption and label are. I wonder if instead of caption you could just take in the label and description as options, and render them into the `<caption>` element. There'd be more logic needed in the template for the conditionals though.